test/main.js

```diff
@@ -17,12 +17,15 @@ import edge from 'selenium-webdriver/edge'
 let config
 const browsersFailures = {}
 let totalFailures = 0
+export let localhostUrl
 
 if (!process.env.CONFIG_FILE) {
   console.error('INFO: CONFIG_FILE not set, so using the default config.json')
   config = require('./config')
+  localhostUrl = config.localhostUrl
```

Comment:
IMO it's less confusing to assign specific variable names for each of these config instances rather than use a generic `config` variable name. So in this case, I would do this:
```suggestion
  const defaultConfig = require('./config')
  localhostUrl = defaultConfig.localhostUrl
```

Reply:
Noted, I'll make that change.

Reply:
Ok, there is a problem here, it comes down to the way the test runner is setup, it uses the variable ```config```, so using a ```defaultConfig``` and ```envConfig``` will require this to be re-written and I really dont want to touch that thing at all.